docs/config.md

```diff
@@ -414,6 +414,13 @@ The following settings are available:
 : *New in `nf-azure` version `0.11.0`*
 : Specify the ID of the Compute Node agent SKU which the pool identified with `<name>` supports (default: `batch.node.centos 8`).
 
+`azure.batch.pools.<name>.startTask`
+: *New in `nf-azure` version `1.5.2`*
+: Specify the startTask that is executed as the node joins the Azure Batch node pool (default: `bash -c "chmod +x azcopy && mkdir \$AZ_BATCH_NODE_SHARED_DIR/bin/ && cp azcopy \$AZ_BATCH_NODE_SHARED_DIR/bin/"`).
+
+`azure.batch.pools.<name>.startTaskPrivileged`
+: Enable the task to run with elevated access. Ignored if `runAs` is set (default: `false`).
```

Comment:
would make sense to run it always as privileged ?

Reply:
then, I'd ask to have this organised as 

```
azure.batch.pools.<name>.startTask.script
azure.batch.pools.<name>.startTask.privileged
```

Reply:
it's enough to add a small class 

```
class AzStartTask {
  String script
  boolean unprivilged

  AzStartTask(Map opts) { ... todo }
}
```

and create it using the usual pattern in the config